bootstore/src/schemes/v0/fsm.rs

```diff
@@ -289,16 +290,16 @@ impl Fsm {
     /// be reconstructed.
     pub fn load_rack_secret(&mut self, now: Instant) -> Result<Uuid, ApiError> {
         self.check_init_err()?;
-        let (rack_uuid, threshold) = match &self.state {
+        let pkg = match &self.state {
             State::Uninitialized => return Err(ApiError::NotInitialized),
             State::Learning { .. } => return Err(ApiError::StillLearning),
-            State::InitialMember { pkg, .. } => (pkg.rack_uuid, pkg.threshold),
-            State::Learned { pkg } => (pkg.rack_uuid, pkg.threshold),
+            State::InitialMember { pkg, .. } => &pkg.common,
+            State::Learned { pkg } => &pkg.common,
         };
         let request_id = self.request_manager.new_load_rack_secret_req(
             now,
-            rack_uuid,
-            threshold,
+            pkg.rack_uuid,
```

Comment:
Nit: now that we've got the `SharePkgCommon` type, should we pass `&pkg` into `new_load_rack_secret_req` and `new_learn_received_req`, instead of the individual members?

(I don't feel strongly about this, but proposing as an option)

Reply:
Eh, maybe ?